test/services/database-analyzer/database-analyzer-sequelize.test.js

```diff
@@ -32,18 +37,36 @@ describe('services > database analyser > Sequelize', () => {
       expect(result.customers).toStrictEqual(expected.customers);
     }, TIMEOUT);
 
-    if (dialect === 'postgres') {
-      it('should generate a model with default values', async () => {
-        expect.assertions(1);
+    it('should generate a model with default values', async () => {
+      expect.assertions(1);
+
+      const defaultsValueExpected = {
+        [databaseUrls.DATABASE_URL_POSTGRESQL_MIN]: _.cloneDeep(expectedDefaultValuesPostgres),
+        [databaseUrls.DATABASE_URL_POSTGRESQL_MAX]: expectedDefaultValuesPostgres,
+        [databaseUrls.DATABASE_URL_MYSQL_MAX]: expectedDefaultValuesMysql,
+        [databaseUrls.DATABASE_URL_MSSQL_MIN]: expectedDefaultValuesMssql,
+        [databaseUrls.DATABASE_URL_MSSQL_MAX]: expectedDefaultValuesMssql,
+      };
+
+      defaultsValueExpected[databaseUrls.DATABASE_URL_POSTGRESQL_MIN]
```

Comment:
WHy not defining this before the  test 🤔  ? It seems static to me

Reply:
It is static, I could move it to the top of the file.
I put it there so that readers know for which test we're doing that.